packages/ui/src/components/va-counter/VaCounter.vue

```diff
@@ -19,21 +20,33 @@
             :aria-label="t('decreaseCounter')"
             v-bind="decreaseButtonProps"
             @click="decreaseCount"
+            @mousedown.prevent="startDecreasing"
```

Comment:
It looks like something you want to have in composable:

```ts
const { listeners } = useLongPress({
  start: () => alert(), 
  finish: () => alert(),
})
```

or using useTemplateRef to make it clearer.

```ts
useLongPress('increaseButtonRef', {
  start: () => alert(), 
  finish: () => alert(),
})
```

I'm not sure if we already have this one. Need to search for something like usePressed or useMouse.

This is the way I would do it.

Reply:
Thank you for the feedback! I tried to follow your advice and moved the logic to useLongPress in the compostables. Please let me know if you have any other suggestions or tips for the code!